### ISSUE TITLE
Updated 0.8.1.12 Server Sending Invalid Entity Events

### ISSUE THREAD

**Issue by ZealanL** [Regalis11/Barotrauma#697]
After updating my server to the latest version from the master branch of this Github and testing it out, it turns out it seems to be sending invalid events. 

Niruth informed me that the source on master is multiple steps ahead of the packaged release, but I see no new commits relating the entity events and event networking since the 0.8.1.12 release.

Were there any changes to the way entity events are sent in the commits since the 0.8.1.12 release?

**Comment by Regalis11** [comment]
There have been no changes to the way entity events are sent, but there's are a bunch of other changes that will cause problems if the clients are not also using the updated version. For example commits 45939a9, bcd44f7, 7a7c92a, 6cb6e26 and 4866a89.

**Comment by NilanthAnimosus** [comment]
Yeah you want to avoid using commits above the latest released version when they alter the networking - I tend to use some of them, such as 45939a91440f1a3cd0ef2bc7596e0a4755fc725d to fix bugs for the server-side parts of it, but if it needs client changes its best avoided like the plague.

6cb6e26b94c09cde89e6339827add382dd6e8401 is actually a safe one I already applied - if the property isn't in the list it should just be ignored I think, and no vanilla code would trigger that anyways in any bad way.

I did tell you (Not sure its before or after this comment for this issue) that it would expload violently on you if you downloaded the latest git over the specific git version, though without a git client not sure how to download up to the specific commit via the website.
